### ISSUE TITLE
Would your model work with anchor based on positive/negative value?

### ISSUE THREAD

**Issue by kanitw** [vega/vega-label#17]
Like this: 

![image](https://user-images.githubusercontent.com/111269/54078354-80c02f00-427b-11e9-9d5e-e11d41e5a325.png)

**Comment by chanwutk** [comment]
Yes, it would work by setting `anchor: ['top', 'bottom']`. Would this seem hacky?

**Comment by kanitw** [comment]
It's not hacky, but my question is how would the negative one know how to use "bottom" given there is also no collision at the top?

**Comment by chanwutk** [comment]
Actually, it would not work when using bar-mark as the base mark.
It would work when place labels at the end point of each bar, and then use the bar-mark as `avoidMarks`.
